client/src/app/pages/applications/components/application-detail-drawer/application-detail-drawer.tsx

```diff
@@ -194,6 +210,17 @@ const TabDetailsContent: React.FC<{
                 <Link to={getIssuesSingleAppSelectedLocation(application.id)}>
                   Issues
                 </Link>
+                <Text component="small">
+                  {application.tasks.currentAnalyzer === undefined ||
+                  application.tasks.currentAnalyzer.state === "Failed"
```

Comment:
This may need to be a wider check.

Something more like:
https://github.com/konveyor/tackle2-ui/blob/0cfe60b490e386a360a46e38769b56057845d377/client/src/app/pages/applications/components/application-detail-drawer/application-detail-drawer.tsx#L368-L369

Reply:
The changes were made in the commit 'use wider check'